src/utilcode/util.cpp

```diff
@@ -2279,32 +2216,6 @@ HRESULT GetImageRuntimeVersionString(PVOID pMetaData, LPCSTR* pString)
     
     // Header data starts after signature.
     *pString = (LPCSTR) pSig->pVersion;
-    if(*pString) {
```

Comment:
This branch seems like a very old code path that ensures Version strings in metadata are valid. It seems like a legacy code path for early development of the CLR.

@jkotas I debugged through this via crossgen and it basically just validates the embedded version string is acceptable and if not replaced it with `v1.0.3705` 😖. Is there anything more I could do to verify this change is okay?

Reply:
I think deleting this is fine. CoreCLR does not use the metadata string for anything.